bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/MediaControlPlayer.java

```diff
@@ -40,12 +40,12 @@ public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command c
             handler.getSocket().play(getDefaultResponseListener());
         } else if (PlayPauseType.PAUSE == command) {
             handler.getSocket().pause(getDefaultResponseListener());
-        } else if (RewindFastforwardType.FASTFORWARD == command) {
+        } else if (NextPreviousType.NEXT == command) {
             handler.getSocket().fastForward(getDefaultResponseListener());
-        } else if (RewindFastforwardType.REWIND == command) {
+        } else if (NextPreviousType.PREVIOUS == command) {
             handler.getSocket().rewind(getDefaultResponseListener());
```

Comment:
You are calling rewind on the socket but you are handling the previous command. Should you rename the socket method as well?

Reply:
The lg api itself calls it rewind and fastForward. Thus, we should not change the name of the method in socket: 

"ssap://media.controls/fastForward"
"ssap://media.controls/rewind"

I think I tested a quite a while ago if "ssap://media.controls/next" existed, and I think it does not.

Maybe that some app on the TV does interpret fastForward as next, but that is not guaranteed.  Previous and Next could also be equivalent to LEFT or RIGHT to the sendButton action: actions.sendButton("LEFT"). 

Since this interpretation is application specific...

@memphi2 In which app on the TV did you test the behavior?